expr/functions/hitcount/function_test.go

```diff
@@ -55,11 +55,12 @@ func TestHitcount(t *testing.T) {
 					math.NaN(), math.NaN(), math.NaN(), math.NaN(), math.NaN(), math.NaN(),
 					5}, 5, now32)},
 			},
-			[]float64{35, 70, 105, 140, math.NaN(), 25},
+			[]float64{5, 40, 75, 110, 120, 25},
+
 			"hitcount(metric1,'30s')",
 			30,
-			now32,
-			now32 + 31*5,
+			1410344975,
+			1410345155,
```

Comment:
These should hold that end = from +count*step
```suggestion
			1410344975 + 31*5,
```